### ISSUE TITLE
Error parsing "profile-data" directory

### ISSUE THREAD

**Issue by bigrob69** [mbolli/nfsen-ng#12]
Hello 

I am unable to import historic data. The following error message is received.

Processing 1 sour...33.2% 364/1097 ETC: < 1 sec. Elapsed: < 1 sec [=========>                     ]
0.2758s /var/nfdump/profiles-data//*/2016/08/28 does not exist!

The directory structure does however contain the files and directory structure
/var/nfdump/profiles-data/2016/08/01 -> /var/nfdump/profiles-data/2016/08/30 

Please assist

**Comment by mbolli** [comment]
It seems like your sources are not configured correctly. In the profiles-data directory, create a directory for each source. Move the remaining structure (2016/08/...) in there.

**Comment by antman2** [comment]
@mbolli That information seems like something really important to be put in the `settings.php` documentation. I wasted lots of time wondering why `cli.php -f -p -ps import` wasn't importing data from the default `/var/cache/nfdump` directory. With your tip I was able to tweak `/lib/systemd/system/nfdump.service` with:
```
ExecStart=/usr/bin/nfcapd -4 -p 9996 -I inside -l /var/nfdump/profiles-data/inside -S 1 -P /var/run/nfcapd.pid  -D
```
... and finally see data on the charts.
